web/nginx.conf.template

```diff
@@ -63,9 +63,9 @@ http {
       proxy_set_header x-forwarded-for $proxy_add_x_forwarded_for;
       proxy_pass       ${HEADWAY_TILESERVER_URL}/fonts/;
     }
-    location ~ ^/bbox.txt$ {
+    location /bbox.txt {
       add_header Content-Type text/plain;
-      return 200 '${HEADWAY_BBOX}';
+      alias ${HEADWAY_SHARED_VOL}/bbox.txt;
```

Comment:
I like your approach here better since it removes the need for manually moving/updating when operating locally, which is a DevX/OpX win, while keeping dynamic data out of the build. :+1:  

(Another thought I had was to put the `bboxes.csv` data in the web image and read out the right one from there. But that also has some rigidity since with your approach we can specify arbitrary areas so I still think what you're doing here is preferred)

(Just noting that for a more scalable-prod-focused architecture, the opposite tradeoff would usually be preferred to remove the need of mounting a volume but that's not what we're aiming for for now)